scripts/hotfix.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+"""
+Script to generate alton and git commands for executing hotfixes
+Commands for:
+  - cutting amis
+  - creating hotfix tag
+
+The script should be run with the hotfix's git hash as a command-line argument.
+Otherwise, the script will prompt the user for the hotfix hash.
+i.e. `python scripts/hotfix.py <hotfix hash>`
+"""
+import datetime
+import sys
+
+if len(sys.argv) == 2:
+    hotfix_hash = sys.argv[1]
+else:
+    hotfix_hash = raw_input("Please enter hotfix's git hash: ")
```

Comment:
It would probably be better to use the [`optparse`](https://docs.python.org/2/library/optparse.html) or [`argparse`](https://docs.python.org/2.7/library/argparse.html) library for this. If you're using `argparse`, it would look something like:
```python
import argparse

parser = argparse.ArgumentParser(description="Generate alton commands for hotfixes")
parser.add_argument("hash", help="git hash for hotfix")
args = parser.parse_args()
print(args.hash) # this is the parsed hash
```